cmake/BLTInstallableMacros.cmake

```diff
@@ -428,37 +428,80 @@ endmacro(blt_add_target_compile_flags)
 
 
 ##------------------------------------------------------------------------------
-## blt_convert_to_system_includes(TARGET <target>)
+## blt_convert_to_system_includes(TARGETS [<target>...]
+##                                [QUIET]
+##                                [INTRANSITIVE])
 ##
 ## Converts existing interface includes to system interface includes.
 ##------------------------------------------------------------------------------
-macro(blt_convert_to_system_includes)
-    set(options)
+function(blt_convert_to_system_includes)
+    set(options INTRANSITIVE QUIET)
```

Comment:
Instead of `INTRANSITIVE`, I would suggest using the already BLT convention used in `blt_print_target_properties` of `CHILDREN   <TRUE|FALSE>`

Reply:
Thinking about it more, does this even need to be an option? Is there ever a case where you'd want a top level library's headers to be treated as system includes but not the headers of its dependencies?

Reply:
I'm now using CHILDREN, but the quest still remains as to whether it is even necessary to make an option.

Reply:
There are some use cases for it, so I'm leaving in CHILDREN as a parameter.